spec/support/initializers/capybara.rb

```diff
@@ -28,3 +28,22 @@
     driven_by :headless_chrome
   end
 end
+
+# adapted from <https://medium.com/doctolib-engineering/hunting-flaky-tests-2-waiting-for-ajax-bd76d79d9ee9>
+def wait_for_javascript
+  max_time = Capybara::Helpers.monotonic_time + Capybara.default_max_wait_time
+  finished = false
+
+  while Capybara::Helpers.monotonic_time < max_time
+    begin
+      finished = page.evaluate_script("initializeBaseApp")
```

Comment:
I chose `initializeBaseApp` because it's the entry point of the client side JS

Reply:
Should we check if the javascript is done running first before we execute the initialize script ourselves? In the past I have used something like this to check for active running javascript. Not sure if something like that is an option here.
```ruby
  def wait_for_ajax
    wait_for { page.evaluate_script("$.active").zero? }
  end
```

Reply:
This doesn't run the initializer, it just checks if `initializeBaseApp` is defined, in previous apps I used to check for things like jQuery like in the blog post mentioned in the description but we don't use it. 

See an example:

```ruby
[1] pry(#<RSpec::ExampleGroups::EditingWithAnEditor>)> page.evaluate_script("initializeBaseApp")
Selenium::WebDriver::Error::JavascriptError: javascript error: initializeBaseApp is not defined
  (Session info: headless chrome=79.0.3945.117)
[2] pry(#<RSpec::ExampleGroups::EditingWithAnEditor>)> page.evaluate_script("window.navigator")
=> {"appCodeName"=>"Mozilla", ... }
```

I'm going to simplify the code removing the error handler, by using `typeof`:

```ruby
[16] pry(#<RSpec::ExampleGroups::EditingWithAnEditor>)> page.evaluate_script("typeof initializeBaseApp")
=> "undefined"
[17] pry(#<RSpec::ExampleGroups::EditingWithAnEditor>)> page.evaluate_script("typeof window.navigator")
=> "object"
```

Reply:
Ah, I get it now! Thank you for rewriting it to be more clear for js noobs like myself 😊